string2int/string2int/string2int.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+
+int* str2int(const char* string)
+{
+	//decleration of index representing the position in the string
+	int i = 0;
+	int Negative = 0;
+
+	if (string[i] == 45)
```

Comment:
what's magic about 45? Move it to a #define with a descriptive name and use that name.